common/artifacts/000_wc_artifacts.txt

```diff
@@ -801,6 +1057,93 @@ dark_crystal = {
 
 	picture = "GFX_inv_misc_orb_03"
 }
+horn_of_cenarius = {
+	quality = 5
+	flags = { misc powerful_magical_artifact }
+
+	active = {
+		is_adult = yes
+		is_druid_class_trigger = yes
+	}
+	allowed_gift = {
+		always = yes
+	}
+
+	stacking = no
+	indestructible = yes
+
+	health = 0.75
+
+	monthly_character_prestige = 0.7
+	monthly_character_piety = 0.4
+
+	druidism_group_opinion = 5
+	dryad_group_opinion = 5
+	night_elf_group_opinion = 5
+
+	text_effect_powerful_magical_artifact = 1
+
+	picture = "GFX_horn_of_cenarius"
+	slot = libram
+}
+shadow_orb = {
+	quality = 5
```

Comment:
Why it's legendary? It stopped appearing in the lore after Warcraft 3. I would give it 3 quality.

Reply:
by value it's right in between legendary and epic.